docs/CONFIGURATION.md

```diff
@@ -324,7 +324,7 @@ Determines whether to enabled the submission to the eqip webservice
 
 **Target** - Back-end (api)<br>
 **Default** - True<br>
```

Comment:
When we say "default" here...default if there's no `.env`?

Reply:
That's correct – it described what the configuration value will be if the software doesn't detect it in the OS environment (for runtime variables) or if it's not in `.env` for the build time ones.